src/server/detail/save_stages_controller.cc

```diff
@@ -113,10 +113,10 @@ GenericError RdbSnapshot::Start(SaveMode save_mode, const std::string& path,
   io_sink_.reset(file);
 
   is_linux_file_ = file_type & FileType::IO_URING;
+  bool align_writes = (file_type & FileType::DIRECT) != 0;
```

Comment:
this is the only functional change - seems we had a bug before and clang warned about it